### PR TITLE
Move project release request modal to its own page

### PR DESCRIPTION
**Pull request by dmarcoux**
preview

### CODE REVIEW

**Comment by dmarcoux** [comment]
[[L] Review modals on Project#show](https://trello.com/c/2hRepM3w/1399-l-review-modals-on-projectshow)

File: src/api/app/views/webui/project/release_request.html.haml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+- @pagetitle = 'Create Maintenance Release Request'
+
+.card.mb-3
```
Comment (hennevogel) [inline src/api/app/views/webui/project/release_request.html.haml:3]: We should not use `card` outside of the grid. The grid is for laying out the page. Cards are for styling content. Build the grid, put the content in it.

Comment (dmarcoux) [reply src/api/app/views/webui/project/release_request.html.haml:3]: So I've came to the realization that we do misuse the `card` component. It's true that we should build the layout with a grid/flexbox. The card component is a container, but it's not meant to be used to create a layout. It's for content and it's pretty clear when looking at the card component documentation in https://getbootstrap.com/docs/4.4/components/card/#content-types

I guess I just didn't know much about the grid/flexbox whenever we started the Bootstrap migration. Then never went back to check what the card component actually is.

Comment (dmarcoux) [reply src/api/app/views/webui/project/release_request.html.haml:3]: For now, we'll create the new views with the grid and eventually, we'll have to refactor our views to stop misusing the card component.

**Pull request merged**